src/urdf_parser_py/urdf.py

```diff
@@ -547,7 +561,8 @@ def from_parameter_server(cls, key='robot_description'):
 
 
 xmlr.reflect(Robot, tag='robot', params=[
-    xmlr.Attribute('name', str, False),  # Is 'name' a required attribute?
+    xmlr.Attribute('name', str),
```

Comment:
Is making the `name` attribute required an intentional change?

Reply:
I did intentionally change it.  The xsd specifies it as required: https://github.com/ros/urdfdom/blob/3ffc4eebb22e24bef503849eb43719a7c148e999/xsd/urdf.xsd#L318 .

In normal circumstances, I wouldn't attempt to change this in an already released distribution like melodic.  But I think this case is somewhat special, because the C++ URDF parser *already* enforces having a name attribute at https://github.com/ros/urdfdom/blob/3ffc4eebb22e24bef503849eb43719a7c148e999/urdf_parser/src/model.cpp#L113 .  Thus, it wouldn't be possible to get too far in the ROS ecosystem without a name for the robot.  So I think it is justified to change it here.  Thoughts?